public/default-welcome-room/diograph.json

```diff
@@ -18,6 +18,8 @@
       }
     },
     "modified": "2020-01-03T14:03:04.751Z",
-    "text": "Welcome to Diory"
+    "text": "Welcome to Diory",
+    "latitude": 60.01366036242365,
+    "longitude": 20.007133483886722
```

Comment:
Every new Diory user starts his/her journey from a lake in Åland :D 

![image](https://user-images.githubusercontent.com/1319383/99580458-27b11400-29e8-11eb-9ab5-9f5f7611527b.png)

Reply:
Good that you added the location so that map lens is useful there too!

What about timeline? Does it show something on the welcome screen?

Reply:
Yeah, it was location bingo! Without a date it should show today. However welcome screen should be thought again.